homeassistant/components/alarm_control_panel/egardia.py

```diff
@@ -48,77 +27,35 @@
     'UNKNOWN': STATE_UNKNOWN,
 }
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_HOST): cv.string,
-    vol.Required(CONF_PASSWORD): cv.string,
-    vol.Required(CONF_USERNAME): cv.string,
-    vol.Optional(CONF_VERSION, default=DEFAULT_VERSION): cv.string,
-    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-    vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-    vol.Optional(CONF_REPORT_SERVER_CODES): vol.All(cv.ensure_list),
-    vol.Optional(CONF_REPORT_SERVER_ENABLED,
-                 default=DEFAULT_REPORT_SERVER_ENABLED): cv.boolean,
-    vol.Optional(CONF_REPORT_SERVER_PORT, default=DEFAULT_REPORT_SERVER_PORT):
-        cv.port,
-})
+D_EGARDIASYS = 'egardiadevice'
+D_EGARDIADEV = 'egardia_dev'
+D_EGARDIASRV = 'egardiaserver'
+CONF_REPORT_SERVER_CODES_IGNORE = 'ignore'
```

Comment:
Import this from the component. There seems to be a bug in the parsing method in the alarm entity too. It compares something to this constant. It should compare to the config entry for this constant.

Reply:
Import done. The bug I assume you found is in parsestatus - however, that is not a bug, this is as I need it to work. Basically the `lookupstatusfromcode` function will return 'ignore' if the status needs to be ignored (based on the config), dus if the status is equal to `CONF_REPORT_SERVER_CODES_IGNORE` (which is `ignore`) then the new status is ignored.

Reply:
Ok, but then please use another constant that doesn't start with `CONF_`. Those should only be used for config keys.

Reply:
Fair enough, I made the change.